### PR TITLE
Markdown File for Style Guide in #120

### PR DESCRIPTION
**Pull request by katrinabrock**
Changed only formatting and wording from @sgrif 's comment on #120 . Content is the same. Once this finds a permanent home, we can link to it from the contrib guide.

### CODE REVIEW

File: guide_drafts/style_guide.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    1,
+    2,
+    3
+]
+```
+
+### Keep lines under ~80 characters
+
+For code, this is a soft guideline. You don't need to break it up when it's 81.
+A rough guideline is \"Does this fit in my terminal pane?\",
```
Comment (killercup) [inline guide_drafts/style_guide.md]: Why the escaped quotes?

File: guide_drafts/style_guide.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+which for most of us comes out to roughly 80ish characters,
+but there's no _hard_ character limit.
+
+Documentation should have a _hard_ wrap at 80 characters.
```
Comment (killercup) [inline guide_drafts/style_guide.md]: Hard wrap at 80 cols _in total_ or hard wrap at "indentation + `/// ` + 80 cols of text"? I think rustfmt uses the latter (or actually the minimum of the latter and 100 cols)

File: guide_drafts/style_guide.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+// ALSO BAD
+fn foo(some_args: OmgVerboseTypeName, more_args: OmgVerboseTypeTwo)
+    -> OurReturnType {
+        // ...
+    }
+```
+
+Why?
+While it costs vertical space,
+the opening bracket visually separates the function signature and the fuction body
```
[the remark this answers is not in the document]
Comment (Eijebong) [reply guide_drafts/style_guide.md]: What about `the function signature and its body` ?

**Comment by sgrif** [comment]
A lot of this is really out of date now that we're using `rustfmt`.

**Comment by killercup** [comment]
Oh, I thought this was already the style we use with rustfmt… that's why I didn't really question anything! 😅 Are there any significant changes? One I know of is the `where` on its own line.

**Comment by katrinabrock** [comment]
So I'm closing this w/o merging now based on @sgrif 's  comment. I'm going to write a little burb about rustup in contributing.md instead

**Pull request closed without merging**